cranelift/codegen/src/machinst/lower.rs

```diff
@@ -699,6 +695,11 @@ impl<'func, I: VCodeInst> Lower<'func, I> {
                 self.cur_scan_entry_color = Some(entry_color);
             }
 
+            // Skip lowering branches; these are handled separately.
```

Comment:
doc suggestion: it might be useful to tell the reader where to look for branch lowering. It doesn't need to be too specific to be robust against refactoring but I could use a helpful hint to get me looking in the right place.

Reply:
Updated, thanks!